src/envs/__init__.py

```diff
@@ -51,3 +54,23 @@ def create_env(name: str) -> BaseEnv:
     if name == "repeated_nextto":
         return RepeatedNextToEnv()
     raise NotImplementedError(f"Unknown env: {name}")
+
+
+def create_env(name: str) -> BaseEnv:
+    """Create an environment given its name.
+    """
+    env = _create_new_env_instance(name)
+    _MOST_RECENT_ENV_INSTANCE[name] = env
+    return env
+
+
+def get_env_instance(name: str) -> BaseEnv:
```

Comment:
I'm a bit confused why we need this. Why can't the user of `create_env` just keep track of the returned environment instance themselves, rather than relying on this cache? Would it make more sense to have `create_env` just return what's in the cache if something already exists, rather than overwriting the cache?

Reply:
It's confusing to have `get_env_instance` be able to create a new instance, because that's redundant with the functionality of `create_env`. I'd like to propose renaming `get_env_instance` to `get_cached_env_instance` with the following implementation:
```
assert name in _MOST_RECENT_ENV_INSTANCE
return _MOST_RECENT_ENV_INSTANCE[name]
```

The user would be responsible for making sure they call `create_env` before ever calling `get_cached_env_instance`, and then it's clear that these two methods have non-overlapping functionalities.

Reply:
thanks, the new version looks good. to your first question, that would've been another way to go, but we would need to change `get_gt_nsrts` so that it takes an environment instance as an argument. what we have now seems at least as good to me.